.claude/hookify.no-manual-atlas-sum.md

```
@@ -0,0 +1,19 @@
+---
+name: no-manual-atlas-sum
+enabled: true
+event: edit
+pattern: atlas\.sum
+action: block
+---
+
+**BLOCKED: Do not manually edit atlas.sum.**
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)